### PR TITLE
Add support for `--paths-from-command` to extra borg arguments.

### LINKED ISSUE #1537: FR: Support --paths-from-command or side of file filtering
#### The problem

Currently while vorta allows you to specify command line arguments to borg, the way in which the arguments are ordered means `--paths-from-command` cannot be used. 

Example as per borg docs:
```sh
borg create --paths-from-command /path/to/repo::joes-files -- find /srv/samba/shared -user joe
```

What actually happens:
image

Even when a folder is specified:
image

```
2023-01-09 12:33:57,989 - vorta.borg.borg_job - INFO - Running command /usr/local/bin/borg create --list --progress --info --log-json --json --filter=AM -C lz4 --paths-from-command -- find ~/Documents/Backup -type f -size -76M user@client-backup.ais:backup::computer-2023-01-09-123357 /Users/username/a
```

#### Requested Solution
Either allow arguments as a suffix and prefix to the borg command and add option not to check for sources.

#### Alternatives
Add a UI selection for size of files that automatically uses `--paths-from-command` or `--paths-from-stdin` under the hood

#### Additional context
Im trying to deploy vorta to a number of clients and want to be able to apply a restriction on the size of the files that are backed up. Currently I cant do this with vorta

### PR DESCRIPTION
**Pull request by real-yfprojects**
Backups will run although no sources are specified if `--paths-from-command` is supplied. Also arguments after `--` will be appended to the end of the command after all other arguments. Closes #1537.

* src/vorta/borg/create.py

### CODE REVIEW

File: src/vorta/borg/create.py [hunk 1]
```diff
@@ -77,7 +78,20 @@ def prepare(cls, profile):
...
+        extra_cmd_options = []
+        suffix_command = []
+        if profile.repo.create_backup_cmd:
+            r: List[str] = profile.repo.create_backup_cmd.split('--')
+            if len(r) > 1:
+                s1, *r = r
+                suffix_command = ('--' + '--'.join(r)).split()
+            extra_cmd_options = s1.split()
+
+        if n_backup_folders == 0 and not '--paths-from-command':
```
Comment (real-yfprojects) [inline src/vorta/borg/create.py:94]: Actually one cannot specify sources and a command at the same time.

**Review by m3nu — DISMISSED** [review]

**Comment by real-yfprojects** [comment]
Added a test.

**Comment by JSmith-Aura** [comment]
Thank you so much for implementing this! Super useful for me (and I hope other people too!)

**Comment by real-yfprojects** [comment]
@m3nu github wants another review.

**Review by m3nu — APPROVED** [review]

**Pull request merged**